$NetBSD: patch-ac,v 1.10 2008/10/16 12:22:10 drochner Exp $

Fixes
cairo-type1-subset.c:1220:24: error: macro "read" passed 4 arguments, but takes just 3
>From ickle_ on freenode #cairo.

--- src/cairo-type1-subset.c.orig	2008-09-25 11:36:35.000000000 +0000
+++ src/cairo-type1-subset.c
@@ -1215,7 +1215,7 @@ cairo_type1_font_subset_generate (void  
     }
 
     if (font->face->stream->read) {
-	ret = font->face->stream->read (font->face->stream, 0,
+	ret = (* font->face->stream->read) (font->face->stream, 0,
 					(unsigned char *) font->type1_data,
 					font->type1_length);
 	if (ret != font->type1_length) {
